src/databricks/labs/remorph/install.py

```diff
@@ -0,0 +1,202 @@
+from databricks.labs.blueprint.installation import Installation
+from databricks.labs.blueprint.installer import InstallState
+from databricks.labs.blueprint.tui import Prompts
+from databricks.labs.blueprint.wheels import ProductInfo
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.errors import NotFound
+from databricks.sdk.retries import retried
+
+from databricks.labs.remorph.__about__ import __version__
+from databricks.labs.remorph.config import MorphConfig
+
+logger = logging.getLogger(__name__)
+
+PRODUCT_INFO = ProductInfo(__file__)
+
+
+class WorkspaceInstaller:
+    def __init__(self, prompts: Prompts, installation: Installation, ws: WorkspaceClient):
+        if "DATABRICKS_RUNTIME_VERSION" in os.environ:
+            msg = "WorkspaceInstaller is not supposed to be executed in Databricks Runtime"
+            raise SystemExit(msg)
+        self._ws = ws
+        self._installation = installation
+        self._prompts = prompts
+        self._catalog_setup = CatalogSetup(ws)
+
+    def run(self):
+        logger.info(f"Installing Remorph v{PRODUCT_INFO.version()}")
+        config = self.configure()
+        workspace_installation = WorkspaceInstallation(
+            config,
+            self._installation,
+            self._ws,
+            self._prompts,
+            verify_timeout=timedelta(minutes=2),
+        )
+        workspace_installation.run()
+
+    def configure(self) -> MorphConfig:
+        try:
+            return self._installation.load(MorphConfig)
+        except NotFound as err:
+            logger.debug(f"Cannot find previous installation: {err}")
+        logger.info("Please answer a couple of questions to configure Remorph")
+
+        source_prompt = self._prompts.choice("Select the source", ["snowflake", "tsql"])
+        source = source_prompt.lower()
+
+        skip_validation = self._prompts.confirm("Do you want to Skip Validation")
+
+        catalog_name = self._prompts.question("Enter catalog_name")
+
+        try:
+            self._catalog_setup.get(catalog_name)
+        except NotFound:
+            self.setup_catalog(catalog_name)
+
+        schema_name = self._prompts.question("Enter schema_name")
+
+        try:
+            self._catalog_setup.get_schema(f"{catalog_name}.{schema_name}")
+        except NotFound:
+            self.setup_schema(catalog_name, schema_name)
+
+        config = MorphConfig(
+            source=source,
+            skip_validation=skip_validation,
+            catalog_name=catalog_name,
+            schema_name=schema_name,
+            sdk_config=None,
+        )
+
+        ws_file_url = self._installation.save(config)
+        if self._prompts.confirm("Open config file in the browser and continue installing?"):
+            webbrowser.open(ws_file_url)
+        return config
+
+    @retried(on=[NotFound], timeout=timedelta(minutes=5))
+    def setup_catalog(self, catalog_name: str):
+        allow_catalog_creation = self._prompts.question(
```

Comment:
```suggestion
        allow_catalog_creation = self._prompts.confirm(
```
Confirm returns boolean, question returns string

Reply:
Done